wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -431,54 +455,75 @@ application. For example, here's a typical error which might be issued:
 $deErrorExample
 ```
 
-### Existing wallet errors
+Existing wallet errors
+======================
 
 $deWalletErrorTable
 
+
+Mnemonic Codes
+--------------
+
+The full list of accepted mnemonic codes to secure a wallet is defined by the BIP-39
+specifications and available [here](https://github.com/bitcoin/bips/blob/master/bip-0039/english.txt).
```

Comment:
Let's add a caveat here: it's not _enough_ to pick 12 random words from that list, but obviously the entire BIP-39 algorithm must be followed to generate entropy such that 12 random words from that list can be derived. I was myself guilt in the past for this false belief and I have reasons to believe other users will be hit by the same pitfall. Better be explicit 😁